translations.yaml

```diff
@@ -1,2 +1,9 @@
 theme.page.previousButton: "Previous: {{label}}"
-theme.page.nextButton: "Next: {{label}}"
\ No newline at end of file
+theme.page.nextButton: "Next: {{label}}"
+theme.page.lastUpdated.timeago: Last updated
+theme.markdown.editPage.text: Edit
+theme.toc.header: On this page
+theme.search.navbar.label: Search
+theme.search.recent: Recent searches
+theme.feedback.settings.label: Was this helpful?
+theme.footer.copyrightText: © 2024 XRP Ledger. Open Source.
```

Comment:
These should be omitted so that they can continue to use the default value.

Reply:
Commented out to help in adding languages.